vendordiff.patch

```diff
@@ -1,25 +0,0 @@
-diff -Naupr -x BUILD -x 'AUTHORS*' -x 'CONTRIBUTORS*' vendor/k8s.io/apiserver/pkg/storage/etcd/testing/utils.go /tmp/gopath.qtqLRB/src/k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/storage/etcd/testing/utils.go
```

Comment:
what is this file?

Reply:
Thats a leftover from https://github.com/kubernetes/kubernetes/blob/7f23a743e8c23ac6489340bbb34fa6f1d392db9d/hack/verify-godeps.sh#L109.  The latest `gitignore` has it. But, was not present in my tree, so got added in between. Trying to remove it from index :)

Reply:
Done.